main/src/main/scala/sbt/Defaults.scala

```diff
@@ -1910,6 +1911,19 @@ object Defaults extends BuildCommon {
         }
       )
     )
+
+  def dependencyResolutionTask: Def.Initialize[Task[DependencyResolution]] =
```

Comment:
Could we move LibraryManagement out of Internal and make it public? I like having as little as possible in Defaults.scala.

Reply:
For pure code I'd agree that they should all get kicked out. For tasks and settings, I think it makes sense to include any tasks a plugin author may use in `Defaults.scala` so they would understand what transitive tasks are involved. This is especially true of dynamic tasks like this one since reading the code might give clue for debugging the behavior.